modular_zubbers/code/game/machinery/vending/multisec.dm

```diff
@@ -0,0 +1,87 @@
+/obj/machinery/vending/wardrobe/sec_wardrobe
+	name = "\improper MultiSec Outfitting Station"
+	desc = "A vending machine stocked with a multitude of security outfits, including standardized uniforms in multiple colors and general equipment. Now with more Multi!"
+	icon = 'modular_zubbers/code/game/machinery/vending/multisec.dmi'
+	light_mask = "sec-light-mask"
+	icon_state = "multisec"
+	product_ads = "Beat perps in style!;The stains wash right out!;You have the right to be fashionable!;Now you can be the fashion police you always wanted to be!"
+	vend_reply = "Good luck, Peacekeeper!"
+	product_categories = list(
```

Comment:
```suggestion
/obj/machinery/vending/wardrobe/sec_wardrobe
	name = "\improper MultiSec Outfitting Station"
	desc = "A vending machine stocked with a multitude of security outfits, including standardized uniforms in multiple colors and general equipment. Now with more Multi!"
	icon = 'modular_zubbers/code/game/machinery/vending/multisec.dmi'
	icon_state = "multisec"
	products = list() // Overriding this to the default to prevent any issues when building the product menu
	product_categories = list(
```

Reply:
This has been applied, though I re-added the light_mask as it was a simple text change to multisec-light-mask for it. I forgot to do it when I first made the change, but the actual DMI file I did actually re-name it on, so now it will work once I sync it.